components/resources/wallet_strings.grdp

```diff
@@ -665,10 +665,27 @@
   <message name="IDS_BRAVE_WALLET_SHOW_SPAM_NFTS_DESCRIPTION" desc="Show spam NFTs filter description">Show spam NFTs</message>
   <message name="IDS_BRAVE_WALLET_PORTFOLIO_SETTINGS" desc="Portfolio settings label">Portfolio settings</message>
   <message name="IDS_BRAVE_WALLET_ACCOUNT_FILTER_ALL_ACCOUNTS" desc="Account Filter Selector All Accounts">All accounts</message>
+  <message name="IDS_BRAVE_WALLET_GET_HELP" desc="Get help button">Get help</message>
+  <message name="IDS_BRAVE_WALLET_TRANSACTION_TAKING_LONG_TIME" desc="A transaction is taking a long time label">Taking longer than expected?</message>
+  <message name="IDS_BRAVE_WALLET_VIEW_IN_ACTIVITY" desc="View in activity tab button">View in activity</message>
+  <message name="IDS_BRAVE_WALLET_SAFELY_DISMISS_WINDOW" desc="You can safely dismiss this window message">You can safely dismiss this window.</message>
+  <message name="IDS_BRAVE_WALLET_SENDING_AMOUNT_TO_ACCOUNT" desc="Transaction intent sending amount to account label">Sending <ph name="AMOUNT_BEFORE">$1</ph><ph name="AMOUNT"><ex>20 ETH</ex>$5</ph><ph name="AMOUNT_AFTER">$2</ph> to <ph name="ACCOUNT_BEFORE">$3</ph><ph name="ACCOUNT"><ex>Ethereum Account 2</ex>$6</ph><ph name="ACCOUNT_AFTER">$4</ph></message>
+  <message name="IDS_BRAVE_WALLET_AMOUNT_SENT_TO_ACCOUNT" desc="Transaction intent amount sent to account label"><ph name="AMOUNT_BEFORE">$1</ph><ph name="AMOUNT"><ex>20 ETH</ex>$5</ph><ph name="AMOUNT_AFTER">$2</ph> has been sent to account <ph name="ACCOUNT_BEFORE">$3</ph><ph name="ACCOUNT"><ex>Ethereum Account 2</ex>$6</ph><ph name="ACCOUNT_AFTER">$4</ph></message>
+  <message name="IDS_BRAVE_WALLET_SWAPPING_AMOUNT_TO_ACCOUNT_ON_NETWORK" desc="Transaction intent swapping amount to account on network label">Swapping <ph name="AMOUNT_BEFORE">$1</ph><ph name="AMOUNT"><ex>20 ETH</ex>$5</ph><ph name="AMOUNT_AFTER">$2</ph> to <ph name="ACCOUNT_BEFORE">$3</ph><ph name="ACCOUNT"><ex>Ethereum Account 2</ex>$6</ph><ph name="ACCOUNT_AFTER">$4</ph> on <ph name="NETWORK"><ex>Ethereum Mainnet</ex>$7</ph></message>
+  <message name="IDS_BRAVE_WALLET_AMOUNT_ADDED_TO_ACCOUNT" desc="Transaction intent amount added to account label"><ph name="AMOUNT_BEFORE">$1</ph><ph name="AMOUNT"><ex>20 ETH</ex>$5</ph><ph name="AMOUNT_AFTER">$2</ph> has been added to your account <ph name="ACCOUNT_BEFORE">$3</ph><ph name="ACCOUNT"><ex>Ethereum Account 2</ex>$6</ph><ph name="ACCOUNT_AFTER">$4</ph></message>
```

Comment:
Maybe "**The amount of** ... **has** been added to your account ..."? Otherwise need to deal with **has/have**.